### PR TITLE
Added SmallDev.tools

### PR DESCRIPTION
**Pull request by twf-nikhila**
Added new web development-related tools.

https://smalldev.tools/

### CODE REVIEW

**Comment by SansGuidon** [comment]
Hi Nikhil and thanks for the suggestion ! 
Before adding tools, I like to get a feeling of if that is a tool I would use myself on a daily basis and whose UX fits me. 
I'm a bit troubled with a few things : 
- The tools lack documentation/examples on how to use them. Of course you could say those tools are pretty obvious, but still I feel it would create a better UX and without examples, it is hard to trust a tool to be a right fit without testing with each case (because the, we have to find samples/data ourselves...)
- The UX is not really user friendly, lot of page refreshes, frequent pop-up... and the tools are not really sorted or arranged in a logical order. 
- I feel the list of tools is nice but really oriented to specific use cases and I would miss for instance categories and coverage for more languages than just JS, SQL and HTML.

So TLDR I would conclude by saying it's probably a list of tools made to fit a particular user needs but it's too opinionated for me to fit most user needs.
I hope the tool will keep getting more features and polishing, then I would probably consider including it ! :-)
